### PR TITLE
Recording UI, part 1

### PR DESCRIPTION
**Pull request by Xelowak**
I added a Profile8 after the passions page. This activity explains that the user will need to record an audio file. After pressing continue, the recordingActivity is launched.

### CODE REVIEW

**Review by r-ismer — APPROVED** [review]
LGTM !

**Review by Lollipoke — APPROVED** [review]
Seems good to me

File: app/src/main/res/layout/set_profile_8.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.167" />
+
+    <ProgressBar
+        android:id="@+id/progressBar_p8"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="413dp"
+        android:layout_height="62dp"
+        android:progress="89"
+        app:layout_constraintBottom_toTopOf="@+id/text_p2"
```
Comment (cgntnr) [inline app/src/main/res/layout/set_profile_8.xml]: This seems to come from a copy-paste of another class, should be removed from constraints.

Comment (cgntnr) [reply app/src/main/res/layout/set_profile_8.xml]: Seems to be fixed! All good.

**Pull request merged**